test/services/PaginatedSpec.scala

```diff
@@ -25,27 +25,29 @@ import utils.SpecBase
 import viewmodels.Paginated
```

Comment:
Please replace links(0) with links.head.

Also, please give a line break before the first assertion to increase the readability, 

for example for the test 
"In case of 2 pages, and we are on the 1st, display: 1,2,Next"
give a line break before below line 

paginatorParser.description mustBe "Showing 1 – 25 of 40 approved accounts"